docs/json/radarr/cf/x266.json

```diff
@@ -0,0 +1,24 @@
+{
+  "trash_id": "390455c22a9cac81a738f6cbad705c3c",
+  "name": "x266",
+  "includeCustomFormatWhenRenaming": false,
+  "specifications": [{
```

Comment:
Formatting nitpick 😅 
You probably took it from the x265 CF so I guess that one would need it as well